src/index.js

```diff
@@ -1,25 +1,80 @@
+import '@babel/polyfill';
 import express from 'express';
+import bodyParser from 'body-parser';
 import cors from 'cors';
+import errorhandler from 'errorhandler';
 import morgan from 'morgan';
+import methodOverride from 'method-override';
+import env from './config/env-config';
 import routes from './routes';
 
+const isProduction = env.NODE_ENV === 'production';
+
+// Create global app object
 const app = express();
-const port = process.env.PORT || 3000;
 
 app.use(cors());
+
+// Normal express config defaults
 app.use(morgan('dev'));
-app.use(express.urlencoded({ extended: false }));
-app.use(express.json());
-
-// routes
-app.use(routes);
-app.get('/', (req, res) =>
-  res.status(200).send({ message: 'welcome to BN: jubilee-team' })
-);
-app.all('*', (req, res) => res.send({ message: 'route not found' }));
-
-app.listen(port, () => {
-  console.info(`Listening on port ${port}`);
+
+app.use(bodyParser.urlencoded({ extended: false }));
```

Comment:
Why need to install new packages to handle what express can handle out of the box?

Reply:
It's been taken out on your request